CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 ## 1.0-32
 
+The latest version of the Vision Teaser App includes improvements in algorithms performance, camera calibration process and lane detection accuracy. Also the new version contains new improved ML-models for CV tasks like detection, segmentation and classification. The full list of changes:
+`MapboxVision` v0.12.0
 - Added japanese traffic signs
+- Improved preformance
```

Comment:
typo : `preformance`

Reply:
Also IMO sounds vary vague

Reply:
changed to `Improved performance by utilizing ARM Advanced SIMD architecture extension`, is that better wording?

Reply:
Well, I would expect not implementation side of the performance improvements but user facing. So if the SIMD optimizations reduced CPU usage probably it's good to mention `Reduced CPU usage of some feature`

Reply:
Wait, this is release notes for teaser, who cares about SIMD and Neon? I think "Overall performance improvements" are good enough. Or if it seems vague, we can remove it completely.

Reply:
I'd remove it completely (even `Performance improvements`). Definitely mentioning SIMD, Neon etc is not needed. I've just speeded up existing Neon a bit, it's not noticeable at all from user side.

Reply:
ok, removed